docs/modules/getting_started/examples/code/nifi.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: nifi.stackable.tech/v1alpha1
+kind: NifiCluster
+metadata:
+  name: simple-nifi
+spec:
+  version: 1.16.3-stackable0.1.0
```

Comment:
Would the image tag also be a candidate for templating?

Reply:
Not required here i would say

Reply:
ok